### PR TITLE
[Bug Fix]LPS-148969 

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
@catarineaguiar Please help me scan test results.

Thanks.
Tina.

**Comment by catarineaguiar** [comment]
@tinatian no regressions found.

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina.

**Comment by tinatian** [comment]
FYI @ericyanLr

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/117167

**Comment by shuyangzhou** [comment]
ci:close

File: portal-impl/src/com/liferay/portlet/internal/InvokerPortletImpl.java [hunk 1]
```diff
@@ -195,15 +193,15 @@ public Portlet getPortlet() {
 
 	@Override
 	public ClassLoader getPortletClassLoader() {
-		ClassLoader classLoader =
-			(ClassLoader)_liferayPortletContext.getAttribute(
-				PluginContextListener.PLUGIN_CLASS_LOADER);
```
Comment (ngriffin7a) [inline portal-impl/src/com/liferay/portlet/internal/InvokerPortletImpl.java:200]: @tinatian Does this mean that the {{PluginContextListener.PLUGIN_CLASS_LOADER}} constant is no longer referenced anywhere in the codebase? Does it serve a purpose anymore? Thanks, Neil

Comment (tinatian) [reply portal-impl/src/com/liferay/portlet/internal/InvokerPortletImpl.java:200]: @ngriffin7a I think we should clean it up, but for now there are still usages of it. I will put this into my list

**Pull request closed without merging**